docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

```diff
@@ -519,19 +521,24 @@ PostgreSQL configuration is divided into two different subsections:
 
 These options are mutually exclusive, and can be turned on or off using the _enabled_ option. Exactly one of these two sections must have enabled = true, while the other must have enabled = false.
 
+Note that you can't change the `useExisting` setting after installation.
+
 **Install options**
 
 | Option | Description |
 | :--- | :--- |
-| `port` | Port PostgreSQL is listening to.
-| `restartSecond` | Wait time to restart PostgreSQL if the service crashes.
-| `locale` | locale is used during initialization of the db.
+| `port` | Port PostgreSQL is listening to. |
```

Comment:
There's an "enabled" boolean field for both the install options and useExisting options. https://github.com/yugabyte/yugabyte-db/blob/d6df95227d59cca5950f66f027372a4fc244564b/managed/yba-installer/yba-ctl.yml.reference#L152 This is what they can't change post install, and we should do the same star or italicize in the table and then point them to the note that says those fields can't be changed.